benches/main.rs

```diff
@@ -88,11 +88,11 @@ fn list_int_python_isinstance(bench: &mut Bencher) {
     Python::with_gil(|py| {
         let (validator, input) = list_int_input(py);
         let input = black_box(input.as_ref(py));
-        let v = validator.isinstance_python(py, input, None, None).unwrap();
+        let v = validator.isinstance_python(py, input, None, None, None).unwrap();
         assert!(v);
 
         bench.iter(|| {
-            let v = validator.isinstance_python(py, input, None, None).unwrap();
+            let v = validator.isinstance_python(py, input, None, None, None).unwrap();
```

Comment:
Just a note: having all of these arguments gets a tad confusing, it might be nice to fold everything into a struct kind of how we do with the `Extra` struct.